apps/gp2-server/test/fixtures/working-group.fixtures.ts

```diff
@@ -1,12 +1,15 @@
 import type { gp2 } from '@asap-hub/model';
 import { FetchWorkingGroupsQuery } from '../../src/autogenerated-gql/graphql';
 
-export const getWorkingGroupDataObject = (): gp2.WorkingGroupDataObject => ({
-  id: 'working-group-id-1',
+export const getWorkingGroupDataObject = (
+  overrides: Partial<gp2.WorkingGroupDataObject> = {},
```

Comment:
not a huge fan of those overrides, i'd just have a simple fixture with a single member and then i would modify the output of this factory when necessary

Reply:
done